.github/workflows/deploy-frontend-pr-previews.yml

```diff
@@ -315,6 +291,8 @@ jobs:
           cd platform/
           docker build --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/next-platform-pr-previews:${{ github.sha }} .
           docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/next-platform-pr-previews:${{ github.sha }}
+          docker tag gcr.io/airqo-250220/pr-previews/next-platform-pr-previews:${{ github.sha }} gcr.io/airqo-250220/pr-previews/next-platform-pr-previews:latest
```

Comment:
Why are we letting of of using secrets?

Reply:
Hi @Baalmart 
The image url is not really a secret, or worth being added as a secret.
1. These urls are already in all the k8s manifest files.
2. They are private
3. None owners still can not access them without the required credentials.

Reply:
Thanks @123MwanjeMike for the explanations.

My guess is that we are still following through with the suggestions from the recent discussion about using "global variables"?